README.md

```diff
@@ -65,8 +65,28 @@ var loader = new RenderExtensionLoader({
 })
 ```
 
+### Getting context and assets only
+
+You can get assets and context without acctually fetching files in case you want only the runtime context before loading extension points.
```

Comment:
```suggestion
You can get assets and context without actually fetching files in case you want only the runtime context before loading extension points.
```